woocommerce/woocommerce-android
# Promote extensions Analytics

Closes: #9550 

### Why
As part of the product extension project we added a explore option to the product types filter whose correspondent extension is not yet installed on the store.

This button will redirect the user to the extension's webpage with the idea of promoting it.

### Description
This PR adds the analytic event to track the merchant's interaction with the new explore option in the product filter

Event registration: https://github.com/Automattic/tracks-events-registration/pull/1983

### Testing instructions

Prerequisites: **You must run this test on a site that does not have at least one of the following extensions installed: Subscription, Product bundles, or Composite products.**

1. Open the products tab
2. Select Filters -> Product type
3. Tap on one of the Explore! options and check that the event is tracked

### Images/gif
```
🔵 Tracked: product_filter_list_explore_button_tapped, Properties: {"type":"subscription","blog_id":222408976,"is_wpcom_store":true,"was_ecommerce_trial":true,"plan_product_slug":"wooexpress-small-bundle-yearly","is_debug":true,"site_url":"https:\/\/woo-practically-automatic-witch.wpcomstaging.com"}
```
